echo.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"schneider.vip/problem"
+)
+
+type Echo struct {
+	Message   string `json:"message"`
+	Timestamp string `json:"timestamp"`
+}
+
+// NewEcho creates and returns a new Echo struct
+func NewEcho(message, timestamp string) *Echo {
```

Comment:
I don't think this is used anywhere; You can probably delete this

Reply:
woops...yeap deleted